### PR TITLE
Improve readme and documentation

### PR DESCRIPTION
**Pull request by RX14**
Hi!

I found this repository and idea, and I thought it was a really good idea! But since there was no link to debug-edge.io, I didn't find the full documentation and only had `.images/debug_edge_v1.png`, which didn't mention UART, so I started designing my own pinout with RX/TX pins instead of TRACE, because I thought this standard didn't help me! But, luckily I eventually found your full website where the rationale and full pinout was listed, realised your design was better, and switched back!

I realised that this oopsie would have been avoided if there was better access to the documentation from this repo, since it's probably the main jumping in point of people looking at DebugEdge. So I've removed the old image from the repo, in favour of a link to the more up to date one on the website, and made sure to link to the website. Would be nice to get the repo's URL changed to debug-edge.io too, so it's even harder to miss!

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 # Debug Edge
 A small bridge board to link a programmer to a PCB, via a card edge connector.
+See [debug-edge.io](https://debug-edge.io/) for more the full specification and rationale.
+
+![A 3d render of a completed DebugEdge PCB](DebugEdge.png)
```
Comment (nitz) [inline README.md:5]: Could you please add a double space at the end of the line here, so it puts the next image after a line break?

**Review by nitz — COMMENTED** [review]
Looks awesome to me! Could you stick a double space after the render image so it bumps the spec image to the next line? 🙂

File: README.md (hunk 1 above)
Comment (RX14) [reply README.md:5]: Done (added a newline)

**Comment by nitz** [comment]
Excellent! Thank you so much!

**Comment by tannewt** [comment]
Thanks! I've added the URL to the repo settings as well.

**Pull request merged**